django_tables2/columns/base.py

```diff
@@ -282,33 +282,42 @@ def attrs(self):
         # Update attrs to prefer column's attrs rather than table's
         attrs.update(dict(self.column.attrs))
 
-        # Column ordering class names
-        ordering_class = attrs.get('th', {}).get('_ordering', {})
-
         # Find the relevant th attributes (fall back to cell if th isn't
         # explicitly specified).
         attrs['th'] = AttributeDict(attrs.get('th', attrs.get('cell', {})))
         attrs['td'] = AttributeDict(attrs.get('td', attrs.get('cell', {})))
 
-        # make set of existing classes.
-        th_class = set((c for c in attrs['th'].get('class', '').split(' ') if c))
-        td_class = set((c for c in attrs['td'].get('class', '').split(' ') if c))
+        # Override/add classes
+        attrs['th']['class'] = self.get_th_class_name(attrs['th'])
+        attrs['td']['class'] = self.get_td_class_name(attrs['td'])
+
+        return attrs
+
+    def get_td_class_name(self, td_attrs):
+        """
+        Returns the HTML class attribute for a data cell in this column
+        """
+        classes = set((c for c in td_attrs.get('class', '').split(' ') if c))
+        td_classes = self.table.get_column_class_names(classes, self)
+        return ' '.join(sorted(td_classes))
+
+    def get_th_class_name(self, th_attrs):
+        """
+        Returns the HTML class attribute for a header cell in this column
+        """
+        classes = set((c for c in th_attrs.get('class', '').split(' ') if c))
+        th_classes = self.table.get_column_class_names(classes, self)
```

Comment:
Since these are now in a separate function, I think we can remove the `th_`-prefix or use that everywhere.